components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/token/AccessTokenIssuer.java

```diff
@@ -113,7 +113,7 @@ public static AccessTokenIssuer getInstance() throws IdentityOAuth2Exception {
      * @throws InvalidOAuthClientException
```

Comment:
Comment should be changed as it doesn't throw `InvalidOAuthClientException` anymore

Reply:
Fix with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/477